services/device-registry-base/src/test/java/org/eclipse/hono/deviceregistry/service/credentials/AbstractCredentialsManagementServiceTest.java

```diff
@@ -88,15 +84,22 @@ public void setNotificationSender() {
     public void testNotificationOnUpdateCredentials(final VertxTestContext context) {
         credentialsManagementService
                 .updateCredentials(DEFAULT_TENANT_ID, DEFAULT_DEVICE_ID, new ArrayList<>(), Optional.empty(), SPAN)
-                .onComplete(context.succeeding(result -> context.verify(() -> {
-                    verify(notificationSender).publish(notificationArgumentCaptor.capture());
-
-                    final var notification = (CredentialsChangeNotification) notificationArgumentCaptor.getValue();
-                    assertThat(notification.getTenantId()).isEqualTo(DEFAULT_TENANT_ID);
-                    assertThat(notification.getDeviceId()).isEqualTo(DEFAULT_DEVICE_ID);
-                    assertThat(notification.getCreationTime()).isNotNull();
+                .onComplete(context.succeeding(result -> {
+                    context.verify(() -> {
+                        verify(eventBus).publish(
+                                eq(NotificationEventBusSupport.getEventBusAddress(CredentialsChangeNotification.TYPE)),
+                                notificationArgumentCaptor.capture(),
```

Comment:
FMPOV the captor should be created (using the specific notification type) in the test case and not as a field of the class